common/app/routes/Challenges/Tool-Panel.jsx

```diff
@@ -110,36 +111,29 @@ export default class ToolPanel extends PureComponent {
           )
         }
         <div className='button-spacer' />
-        <ButtonGroup
-          className='input-group'
-          justified={ true }
-          >
           <Button
-            bsSize='large'
             bsStyle='primary'
-            componentClass='label'
+            className='btn-big btn-block'
```

Comment:
btn-big is one of our custom classes, right?

isn't btn-block added by the prop `block={ true }`?